.github/workflows/s3_upload.yml

```diff
@@ -32,24 +32,15 @@ jobs:
         run: |
           npm run build
 
+      - name: Install AWS CLI
+        run: |
+          curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+          unzip awscliv2.zip
+          sudo ./aws/install
+          aws --version
+
       - name: Deploy ${{ steps.extract.outputs.current-version }}
-        uses: reggionick/s3-deploy@v3
-        with:
-          folder: dist/js
-          bucket: ${{ secrets.S3_BUCKET }}/${{ env.PACKAGE_NAME }}/${{ env.PACKAGE_VERSION }}
-          bucket-region: ${{ secrets.S3_BUCKET_REGION }}
-          dist-id: ${{ secrets.CLOUDFRONT_DISTRIBUTION_ID }}
-          invalidation: ${{ secrets.S3_BUCKET }}/${{ env.PACKAGE_NAME }}/${{ env.PACKAGE_VERSION }}/*
-          no-cache: true
-        env:
-          PACKAGE_VERSION: v${{ steps.extract.outputs.current-version }}
+        run: node scripts/deploy-cdn.js --folder dist/js --distId ${{ secrets.CLOUDFRONT_DISTRIBUTION_ID }} --bucketName ${{ secrets.S3_BUCKET }} --packageName ${{ env.PACKAGE_NAME }} --tagName ${{ env.PACKAGE_VERSION }}
```

Comment:
I think you will need to get the `PACKAGE_VERSION` environment variable from somewhere since we no longer extract it. Maybe move the logic into the deploy script and you can import the `package.json` directly and read the version from there? It should be prefixed with `v` like `v5.0.0`, `v5.0.1`, etc.

Will also need to remove the `Extract version` step since that action is not whitelisted.

Reply:
Done!